### PR TITLE
Add Inpad v0.1.0

### PR DESCRIPTION
**Pull request by Sarah-Seo**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/inpad.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'inpad' do
+  version '0.1.0'
+  sha256 'a66a0be66bc98d31f36232ab9b5ea5a47fdb644e4234673d8ccc26bd637841f1'
+
+  # github.com/Sarah-Seo/Inpad was verified as official when first introduced to the cask
+  url "https://github.com/Sarah-Seo/Inpad/releases/download/v#{version}/Inpad-#{version}.dmg"
```
Comment (miccal) [inline Casks/inpad.rb]: Please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/inpad.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'Inpad'
+  homepage 'https://sarah-seo.github.io/Inpad/'
+  license :unknown
```
Comment (miccal) [inline Casks/inpad.rb]: Please remove the `license` stanza, it has been deprecated.

**Review by miccal — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Ping @vitorgalvao RE https://github.com/caskroom/homebrew-cask/issues/28220.

File: Casks/inpad.rb (hunk 2 above)
Comment (Sarah-Seo) [inline Casks/inpad.rb]: I fixed it but got this message...  What did I miss?

```
$ brew cask audit inpad --download
audit for inpad: failed
 - a license stanza is required (:unknown is OK)
Error: audit failed
```

I'm using Homebrew 1.1.5-127-g538028a Homebrew/homebrew-core (git revision 06623; last commit 2017-01-03) now.

Comment (miccal) [inline Casks/inpad.rb]: If you run `brew cask doctor` what does it say under `==> Homebrew-Cask Version:`?

Comment (Sarah-Seo) [inline Casks/inpad.rb]:
```
==> Homebrew-cask Version:
0.60.0 (git revision a8546; last commit 12 minutes ago)
```

Comment (miccal) [inline Casks/inpad.rb]: Your version of Cask is extremily outdated - your `==> Homebrew-Cask Version:` should more-or-less match the `==> Homebrew Version:`. Please run the command
```
cd "$(brew --repository)"; git fetch; git reset --hard origin/master; brew update --force
```

Comment (Sarah-Seo) [inline Casks/inpad.rb]: Thanks, it worked. 👍

Comment (miccal) [inline Casks/inpad.rb]: You're welcome.

**Review by miccal — APPROVED** [review]

**Comment by vitorgalvao** [comment]
> Ping @vitorgalvao RE https://github.com/caskroom/homebrew-cask/issues/28220.

Consensus so far was 10 stars, so lets merge it.

**Comment by Sarah-Seo** [comment]
Thanks for approving! Have a nice day. 😄

**Pull request merged**